proposals/notifications.md

```diff
@@ -0,0 +1,88 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which is submitted to and resides in an [LDP Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
```

Comment:
If it's vocab-agnostic, how does one know that it's a Notification? (The fact that it resides in an Inbox?) You mention above that the definition is simply "LDP resource, resides in an LDP container". That's a bit broad, right? It'd mean that all resources are notifications.